process_photos.py

```diff
@@ -7,7 +7,9 @@ def get_photo_names(photo_dir: str) -> list[str]:
 
 
 def get_large_photo_names(photo_dir: str) -> list[str]:
-    return [f for f in get_photo_names(photo_dir) if "Small" not in f]
+    list = [f for f in get_photo_names(photo_dir) if "Small" not in f]
+    list.sort()
+    return list
 
```

Comment:
You can do this as
```python
sorted(f for f in get_photo_names(photo_dir) if "Small" not in f)
```
`sorted(list)` creates a new list, `.sort()` sorts an existing one in place

Reply:
Changed :+1: